app/main.py

```diff
@@ -1 +1,19 @@
 # write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    input_command, source_file_path, destination_file_path = command.split()
```

Comment:
Move command.split() to a separate variable because you're duplicating it in line 11